Sources/Diagnostics/DiagnosticsEntry.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  DiagnosticsEntry.swift
+//
+//  Created by Cesar de la Vega on 1/4/24.
+
+import Foundation
+
+protocol DiagnosticsEntry: Codable, Equatable {
+
+    var diagnosticType: String { get }
```

Comment:
We can remove the type and counters/histograms now, that's not going to be used by the backend

Reply:
right, I had this from before and forgot to remove. Thanks for pointing that out